src/conditionalExpressions/task2.js

```diff
@@ -0,0 +1,9 @@
+if (typeof value === "boolean") {
+    console.log(`Variable is boolean : ${value}`);
+} else if (typeof value === "number") {
+    console.log(`Variable is a number. It's square is: ${value}`);
+} else if (typeof value === "string") {
+    console.log(`Variable is a string:${value}`);
+} else {
+    console.log(`Variable type is: ${value}`);
```

Comment:
You're printing the value itself instead of it's type. AC requirement says: "append variable type"